packages/loot-core/src/types/models/reports.d.ts

```diff
@@ -1,14 +1,34 @@
-export type DataEntity = {
-  data: GroupedEntity[];
-  monthData: GroupedEntity[];
-  groupedData: GroupedEntity[];
+export interface CustomReportEntity {
+  reportId?: string;
+  mode: string;
+  groupBy: string;
+  balanceType: string;
+  showEmpty: boolean;
+  showOffBudgetHidden: boolean;
+  showUncategorized: boolean;
+  graphType: string;
+  selectedCategories;
+  filters;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏missing type

Reply:
Good catch. Thanks!